requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
-coveralls
```

Comment:
@pelson Don't we need to keep `coveralls` ? Hence, why this PR is not getting a response from the coveralls bot...